checkov/common/util/ext_argument_parser.py

```diff
@@ -557,3 +557,9 @@ def add_parser_args(self) -> None:
                  "resource code to OpenAI to request remediation guidance. This will use your OpenAI credits. "
                  "Set your number of findings that will receive enhanced guidelines using CKV_OPENAI_MAX_FINDINGS",
         )
+        self.add(
+            "--tool-name",
+            default=None,
```

Comment:
would use `Checkov` here to keep the default the same

Reply:
I have changed the default to Checkov as recommended :)